features/element-hiding.json

```diff
@@ -171,6 +171,10 @@
                 "selector": ".top-banner",
                 "type": "hide-empty"
             },
+            {
+                "selector": "#top-banner",
+                "type": "hide"
```

Comment:
Do we want this to be `hide-empty` so we don't start hiding populated content?

Reply:
yes - change made